src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -871,17 +871,17 @@ public function query_child_records( $args = array() ) {
 	 *
 	 * @return WP_Query|WP_Error|bool
 	 */
-	public function get_child_record_by_status( $status = 'success', $qty = -1 ) {
+	public function get_child_record_by_status( $status = 'success', $qty = -1, array $args = array() ) {
 		$statuses = Tribe__Events__Aggregator__Records::$status;
 
 		if ( ! isset( $statuses->{ $status } ) && 'trash' !== $status ) {
 			return false;
 		}
 
-		$args = array(
-			'post_status'    => $statuses->{ $status },
+		$args = array_merge( $args, array(
```

Comment:
Did we want to override any potentially passed values here or should we use `wp_parse_args`? These two values are passed individually so I'm just commenting as a double-check.

Reply:
we want to override so this is ok.